hack/generator/pkg/astmodel/generation_consts.go

```diff
@@ -8,7 +8,14 @@ package astmodel
 var (
 	CodeGenerationComments []string = []string{
 		// Note that this format is actually an official specification in go:  https://github.com/golang/go/issues/13560
-		"// Code generated by k8s-infra. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
+		"// Code generated by azure-service-operator. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
```

Comment:
IMO we should differentiate between ASO and the tool that generates aso.

Maybe this is ASO-gen or something?

Reply:
Changed to `azure-service-operator-codegen`.